File: 백준/Bronze/2920. 음계/음계.py
a = list(map(int, input().split()))
m = [
  [1,2,3,4,5,6,7,8],
  [8,7,6,5,4,3,2,1]
]
if a==m[0]:
  print("ascending")
elif a==m[1]:
  print("descending")
else:
  print("mixed")

File: 백준/Bronze/2920. 음계/음계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int inCheck = 0;
        int deCheck = 0;
        Scanner sc = new Scanner(System.in);

        for (int i = 1; i <= 8; i++){
            int a = sc.nextInt();
            if(i == a){
                inCheck++;
            }else if(9-i == a){
                deCheck++;
            }
        }
        if (inCheck == 8){
            System.out.println("ascending");
        }else if(deCheck == 8){
            System.out.println("descending");
        }else{
            System.out.println("mixed");
        }
    }
}
